public/.htaccess

```diff
@@ -0,0 +1,13 @@
+# rewrite ^/(.*)/$ /$1 permanent;
+
+# if (!-d $request_filename && !-f $request_filename) {
```

Comment:
What's the deal with this commented-out code?

Reply:
none that is the .htaccess that you use.